src/main/java/dk/aau/p4/abaaja/Main.java

```diff
@@ -23,7 +23,7 @@ public static void main(String[] args) {
         ProblemCollection problemCollection = new ProblemCollection();
 
         // Parse test CharStream
-        ParseTree tree = syntaxPhase( CharStreams.fromString("variable test:NUMBER; \nvariable bing: BOOLEAN; \ntest = 2 + bing;"), problemCollection);
+        ParseTree tree = syntaxPhase( CharStreams.fromString("variable test: BOOLEAN; test = var1 == var2;"), problemCollection);
```

Comment:
```suggestion
        ParseTree tree = syntaxPhase( CharStreams.fromString("variable test:NUMBER; \nvariable bing: BOOLEAN; \ntest = 2 + bing;"), problemCollection);
```

Don't change the string, this will make it much easier to merge with interpreter.